infra/gp-cluster-logging-instance/templates/clusterLogForwarder.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+    - name: loki
+      secret:
+        name: loki-hub-secret
+      type: loki
+      url: 'https://loki.infra.gepaplexx.com'
+  pipelines:
+    - inputRefs:
+        - application
```

Comment:
brauchen wir am hub cluster die application logs? oder reichen uns die infrastructure logs hier aus? soll ja hauptsächlich verwendet werden, wenn auf den clustern selbst gar nichts mehr geht/logging dort nicht mehr erreichbar ist.

Reply:
Stimmt, eigentlich brauchen wir die nicht. Aus einem Sicherheitsstandpunkt würde es aber Sinn ergeben, die Audit Logs hinzuschicken.

Reply:
zu brauche wir die application logs?
Was sind application logs? nur kunden Logs? oder auch unsere "internen applikationen"?
potentiell müssen wir auf diversen namespaces noch das label setzten damit logs in infrastructure landen.

Reply:
Separates Issue für Namespace Labels